velox/expression/tests/ExpressionFuzzerTest.cpp

```diff
@@ -58,7 +58,6 @@ int main(int argc, char** argv) {
       // cardinality passing a VARBINARY (since HLL's implementation uses an
       // alias to VARBINARY).
       "cardinality",
-      "in",
```

Comment:
nit: Have you run fuzzer to verify that enabling "in" wouldn't cause new errors? I wonder whether we should enable it separately later when we're able to solve existing fuzzer-found bugs. (cc @laithsakka )

Reply:
Yes, I did run fuzzer with "in" enabled and fixed issues that I found. All the changes here (including support for timestamp) are to fix fuzzer failures.